MantidQt/CustomInterfaces/src/EnggDiffraction/EnggDiffractionViewQtGUI.cpp

```diff
@@ -1474,54 +1504,141 @@ EnggDiffractionViewQtGUI::splitFittingDirectory(std::string &selectedfPath) {
   return splitBaseName;
 }
 
+void MantidQt::CustomInterfaces::EnggDiffractionViewQtGUI::enableMultiRun(
+    std::string firstRun, std::string lastRun) {
+
+  bool firstDig = isDigit(firstRun);
+  bool lastDig = isDigit(lastRun);
+
+  std::vector<std::string> RunNumberVec;
+  if (firstDig && lastDig) {
+    int firstNum = std::stoi(firstRun);
+    int lastNum = std::stoi(lastRun);
+
+    if ((lastNum - firstNum) > 200) {
```

Comment:
@FedeMPouzols Its slightly slow whem adding more run numbers to list view widget 200/300 times